app/assets/stylesheets/dance_comment.scss

```diff
@@ -0,0 +1,27 @@
+.dance_comment_list li {
+  display: block;
+}
+
+.comment_box {
+  border: 3px solid #000000;
+  margin: 5px;
+  padding: 20px;
+  position: relative;
+  background: white;
+}
+
+.comment_box p {
+  padding-left: 5px;
+}
+
+.comment_box footer ul {
+  position: absolute;
+  right: 5px;
+  bottom: 0;
+}
+
+.comment_box footer li {
+  display: inline;
+  font-size: 12px;
+  padding: 20px 8px;
+}
```

Comment:
There is the potential for Sass nesting the `.comment_box` styling here as well :)